src/m365/spo/commands/field/field-set.ts

```diff
@@ -158,7 +163,7 @@ class SpoFieldSetCommand extends SpoCommand {
       // retrieve column CSOM object id
       const fieldQuery: string = args.options.id ?
         `<Method Id="663" ParentId="7" Name="GetById"><Parameters><Parameter Type="Guid">${formatting.escapeXml(args.options.id)}</Parameter></Parameters></Method>` :
-        `<Method Id="663" ParentId="7" Name="GetByInternalNameOrTitle"><Parameters><Parameter Type="String">${formatting.escapeXml(args.options.name || args.options.title)}</Parameter></Parameters></Method>`;
+        `<Method Id="663" ParentId="7" Name="GetByInternalNameOrTitle"><Parameters><Parameter Type="String">${formatting.escapeXml(args.options.name || args.options.title || args.options.internalName)}</Parameter></Parameters></Method>`;
```

Comment:
There's an odd `args.options.name` here, thats not actually part of the options, lets remove it.